add-to-cart/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@coveo/sample-component-project",
```

Comment:
Was the project created as a default sample to have this name ?

Reply:
Locally this is different. Not sure what happened, never seen this happen before that's weird. Even git lens shows that in the previous commit made two days ago, this text is different. Anyways, I'll check to make sure this is fixed on the next commit.  
<img width="662" alt="Screenshot 2023-03-08 at 11 16 01 AM" src="https://user-images.githubusercontent.com/122834615/223768324-7a8ec614-5052-43d7-80ea-3bd872c2bab8.png">

Reply:
@jantoun-coveo actually yeah there is another package.json file in the components/add-to-cart/src folder and that's the one with the screenshot I sent. This was the command I used to create the component. `npm init @coveo/atomic-component add-to-cart`